src/store/group/types.ts

```diff
@@ -1,6 +1,11 @@
 export interface Group {
     ID?: string;
     Name: string;
-    Peers?: any[];
+    Peers?: GroupPeer[] | string[];
     PeersCount?: string;
+}
+
+export interface GroupPeer {
+    ID: string,
```

Comment:
Codacy has a fix for the issue: [Replace `··ID:·string,` with `ID:·string;`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  ID: string;
```